test/utils/add_proxy_test.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+// npm dependencies
+const {expect} = require('chai')
+const urlParse = require('url').parse
+const proxyquire = require('proxyquire').noPreserveCache()
+
+describe('add proxy test', () => {
+  describe('with FORWARD_PROXY_URL env var', () => {
+    it('url should be proxied with new hostname and port', () => {
+      const forwardProxy = urlParse('http://forward-proxy:8888')
+      process.env.FORWARD_PROXY_URL = forwardProxy.href
+      const addProxy = proxyquire('../../app/utils/add_proxy', {})
```

Comment:
don't think this needs to be proxyquired - you aren't stubbing anything

Reply:
unfortunately it doesn't work if i don't use proxyquired. this test doesn't pass:
```
const {addProxy} = require('../../app/utils/add_proxy')

describe('add proxy test', () => {
  describe('with FORWARD_PROXY_URL env var', () => {
    it('url should be proxied with new hostname and port', () => {
      const forwardProxy = urlParse('http://forward-proxy:8888')
      process.env.FORWARD_PROXY_URL = forwardProxy.href
      const proxiedUrl = addProxy(urlParse('http://localhost:8080'))
      expect(proxiedUrl.hostname).to.equal(forwardProxy.hostname)
      expect(proxiedUrl.port).to.equal(forwardProxy.port)
    })
  })
```
i actually got the idea to use proxyquire from an ex colleague's blog ;) https://medium.com/@maxcbc/mocking-environment-variables-in-node-js-a17a416e127c